front/app/components/Avatar/index.tsx

```diff
@@ -18,6 +16,8 @@ import { getFullName } from 'utils/textUtils';
 
 import messages from './messages';
 
+const FeatureFlag = lazy(() => import('components/FeatureFlag'));
```

Comment:
What is the idea behind dynamically importing all these feature flag components? First of all, we are trying to move away from this component and instead use `useFeatureFlag` instead. Second, the code contained in this component is very very minimal.

I don't know for sure if dynamically importing a component also makes all its children dynamically imported, but I am 80% sure the answer is no. So all the children of the feature flag are stilll going to be imported statically.